metron-stellar/stellar-common/src/main/java/org/apache/metron/stellar/dsl/functions/RestFunctions.java

```diff
@@ -244,15 +244,13 @@ protected Object doGet(RestConfig restConfig, HttpGet httpGet, HttpClientContext
         scheduledFuture.cancel(true);
       }
       int statusCode = response.getStatusLine().getStatusCode();
+      LOG.error("request = {}; response = {}", httpGet, response);
```

Comment:
Is this meant to be set to error log level? Wouldn't debug be more appropriate?

Reply:
Yes this was a mistake.  Fixed in latest commit.